mimikatz/modules/token/list.md

```diff
@@ -1,7 +1,10 @@
 # list
 
-`token::list` lists all tokens on the system. It has the following command line arguments:
+`token::list` lists all tokens on the system.
 
+It has the following command line arguments:
+
+* `/token`: The token to list
 * `/user`: The user token to list
```

Comment:
It'd be awesome to link the arguments to an example in the page so that users can easily browse those
Example below:
```md
* [`/system`](./#system): List only system tokens

[...]

## system

List tokens of local `SYSTEM` account:

[...]
```

Reply:
Implemented in 1d25e40 and ee95ac6.

I could not "compile" the GitBook locally, so I could not try if the links actually work. 

Is there a way to compile it locally to verify changes?

Reply:
no way to compile locally, we're depending on gitbook, which doesn't offer a local compilation
I'm working on that to avoid that in the future, but it's not there yet
the alternative is that I give access to the online gitbook editor, it'll be far better for future contributions of yours, please reach out to me on Discord at `nwodtuhs` and I will give you proper access. For security measures, please comment here your Discord handle's SHA256 (`echo -n 'Handle#1234' | sha256sum` or echo -n 'handle' | sha256sum)

Reply:
OK, good idea.

Here's the hash of my discord handle: `721d2a195ab2a12fb54f4bd8066eb17dd3c57fd688a3903ecc1e755a3511a416`

Reply:
I cross-checked the GitBook and noticed that the links in the GitBook point to the wrong location. The `list` part is not included in the URL.

E.g. they point to `.../mimikatz/modules/token#id` instead of `mimikatz/modules/token/list#id`

I'll try to fix this when I have GitBook access and probably can have a preview.